guides/common/modules/proc_importing-custom-ssl-certificates-by-using-cli.adoc

```diff
@@ -1,25 +1,15 @@
 :_mod-docs-content-type: PROCEDURE
 
-[id="Importing_Custom_SSL_Certificates_{context}"]
-= Importing {customssl} certificates
+[id="importing-custom-ssl-certificates-by-using-cli"]
+= Importing {customssl} certificates by using Hammer CLI
 
+[role="_abstract"]
 Before you synchronize {customcontent} from an external source, you might need to import SSL certificates into your {customproduct}.
```

Comment:
```suggestion
Before you synchronize {customcontent} from an external source, you might need to import SSL certificates into your {Project}.
```

tech-wise, you must import (aka. create) a content credential that you then associate with your product and/or repository.